torch_em/data/datasets/bcss.py

```diff
@@ -0,0 +1,128 @@
+import os
+import numpy as np
+from glob import glob
+from typing import Tuple, List, Optional
+
+import vigra
+
+import torch
+
+import torch_em
+from torch_em.data.datasets import util
+from torch_em.data import ImageCollectionDataset
+
+
+URL = "https://drive.google.com/drive/folders/1zqbdkQF8i5cEmZOGmbdQm-EP8dRYtvss?usp=sharing"
+
+
+# TODO
+CHECKSUM = None
+
+
+class BCSSLabelTrafo():
+    def __init__(
+            self,
+            label_choices: Optional[List[int]] = None
+    ):
+        """Label choices:
+            0: outside_roi; 1: tumor; 2: stroma; 3: lymphocytic_infiltrate; 4: necrosis_or_debris;
+            5: glandular_secretions; 6: blood; 7: exclude; 8: metaplasia_NOS; 9: fat; 10: plasma_cells;
+            11: other_immune_infiltrate; 12: mucoid_material; 13: normal_acinus_or_duct; 14: lymphatics;
+            15: undetermined; 16: nerve; 17: skin_adnexa; 18: blood_vessel; 19: angioinvasion; 20: dcis; 21: other
+        """
+        self.label_choices = label_choices
+
+    def __call__(
+            self,
+            labels: np.ndarray
+    ) -> np.ndarray:
+        """Returns the transformed labels
+        """
+        if self.label_choices is not None:
+            _segmentation = np.zeros(labels.shape)
+            for uq_label in self.label_choices:
+                _segmentation[labels == uq_label] = labels[labels == uq_label]
+            segmentation, _, _ = vigra.analysis.relabelConsecutive(_segmentation.astype("uint64"))
+        else:
+            segmentation, _, _ = vigra.analysis.relabelConsecutive(labels)
```

Comment:
If these are semantic labels then we shouldn't do a relabel!
We should discuss tomorrow.